packages/devtools_app/lib/src/screens/network/network_request_inspector.dart

```diff
@@ -55,7 +55,17 @@ class NetworkRequestInspector extends StatelessWidget {
           if (data is DartIOHttpRequestData) ...[
             _buildTab(tabName: NetworkRequestInspector._headersTabTitle),
             if (data.requestBody != null)
-              _buildTab(tabName: NetworkRequestInspector._requestTabTitle),
+              _buildTab(
+                tabName: NetworkRequestInspector._requestTabTitle,
+                trailing: Row(
```

Comment:
Is this `Row` necessary here?

Reply:
Technically not. The `Row` is used for positioning widgets and can be replaced by an `Align`. I wrote this piece of code similarly to the code below.

Reply:
I'd replace this with an `Align` then (we should probably do that below as well).

Reply:
Thank you for your review. I have fixed this error